src/typeahead/index.tsx

```diff
@@ -293,13 +293,14 @@ export const Typeahead = factory(function Typeahead({
 								onBlur={() => {
 									const { onBlur } = properties();
 
+									onBlur && onBlur();
+
 									if (!strict) {
 										const value = icache.getOrSet('value', '');
 										callOnValue(value);
```

Comment:
surely it's this part you want to remove @rorticus ? so it doesn't `callOnValue` when the typeahead `blur`s?